docs/content/releases.md

```diff
@@ -6,7 +6,39 @@ doctypes: ["concept"]
 toc: true
 docs: "DOCS-616"
 ---
+## NGINX Ingress Controller 3.1.1
+
+04 May 2023
+
+OVERVIEW:
+This release reverts the changes made in 3.1.0 to use sysctls to bind to lower level ports without the NET_BIND_SERVICE capability. It also adds support for serviceNameOverride in the Helm chart, that can be used to override the service name for the NGINX Ingress Controller. This is useful especially during an upgrade from versions prior to 3.1.0, to avoid downtime due to the service name change.
```

Comment:
We should have a clear note here in the release note about how to "that can be used to override the service name for the NGINX Ingress Controller."  Inserted right after this overview.
Even if we outline the detail elsewhere.  Calling that behavior out in an obvious way is important to the issues that were reported.

Reply:
I've added the command